v21.1/deploy-cockroachdb-with-kubernetes-openshift.md

```diff
@@ -79,10 +79,10 @@ This article assumes you have already installed the OpenShift Container Platform
     $ oc get pods
     ~~~
 
-	~~~
-	NAME                                  READY   STATUS    RESTARTS   AGE
-	cockroach-operator-65c4f6df45-h5r5n   1/1     Running   0          51s
-	~~~
+		~~~
```

Comment:
formatting is broken in the preview here, but lgtm otherwise!

Reply:
Thanks for catching this! No idea why the formatting doesn't work this way, but I was able to fix it.